django/db/models/options.py

```diff
@@ -177,6 +177,14 @@ def _map_model_details(self, link):
         return link, model, direct, m2m
```

Comment:
There's `model_label = '%s.%s' % (self.app_label, self.model_name)` in this file which could use `self.label_lower`